src/main/java/com/puppycrawl/tools/checkstyle/api/Violation.java

```diff
@@ -86,6 +88,7 @@
      * Arguments for MessageFormat.
      *
      * @noinspection NonSerializableFieldInSerializableClass
+     * @noinspectionreason we do not serialize this class
```

Comment:
`usage of 'Serializable'  for this api class is considered as mistake now, but we do not break api without good reason`

Reply:
Done